lab_4_retrieval_w_clustering/main.py

```diff
@@ -324,6 +514,15 @@ def calculate_square_sum(self) -> float:
         Returns:
             float: Sum of squares of distance from vector of clusters to centroid.
         """
+        sse = 0.0
```

Comment:
try not creating a new variable, but using `sum()`

Reply:
Tried, but what I did might be shocking (I couldn't find a better way)